acceptance/gossip_peerings.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Peter Mattis (peter.mattis@gmail.com)
+
+package main
+
+import (
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/cockroachdb/cockroach/acceptance/localcluster"
+)
+
+var _ = registerTest("gossip-peerings", gossipPeeringsTest)
+
+func checkGossipNodes(client *http.Client, node *localcluster.Container) int {
+	resp, err := client.Get(fmt.Sprintf("https://%s/_status/gossip", node.Addr("")))
+	if err != nil {
+		return 0
```

Comment:
If we're returning 0 due to an error we should probably log it (at least at V(1), if this is something that is expected to retry frequently).